### ISSUE TITLE
Something's wrong with Iroquois pathfinder

### ISSUE THREAD

**Issue by saamohod** [LoneGazebo/Community-Patch-DLL#4900]
_1. Mod version (i.e Date - 4/23):_11-9

_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description:_  

Please see thread https://forums.civfanatics.com/threads/iroquois-pathfinder-in-forest.638526/

**Comment by ilteroi** [comment]
so what's the expected behavior?

right now, the iroquois ability causes forest to be treated as roads. so you get movement like on a road and that's it.

**Comment by saamohod** [comment]
@ilteroi 
I.e. Trailblazer and Woodsman promotions don't add extra movement in forest for Iroquois, do they? It appears that despite forest being their element "lore-wise" Iroquois scout units are slower in forests/jungle than any other civilization.

**Comment by ilteroi** [comment]
did you even read what i wrote?

**Comment by saamohod** [comment]
>did you even read what i wrote?
Of course!
Iroquois treat forests as roads, I get it. Sadly I couldn't figure out what you statement has to do with Trailblazer effect or namely the lack thereof.

**Comment by MoiMagnus** [comment]
@Ilterol So, if I understand correctly, the problem is that Trailblazer pathfinder go slower on forest roads than on pure forest. Can it be changed, or is road movement too hardcoded?

**Comment by chicorbeef** [comment]
I believe the problem might actually be the fact that Woodsman/Trailblazer I can not stack.

**Comment by ilteroi** [comment]
let me rephrase it. 

is the expectation that all the effects stack up? or that only the best one counts? or should promotions always override the UA?

anyway. that piece of code is called millions of times per turn, so i try to keep it as simple as possible.

**Comment by saamohod** [comment]
@ilteroi 

My personal expectation was that only the best would count (after all they are Iroquois, they can't be the worst performers in forest speed-wise. ). 
On the other hand I have no clue about the coding of the mod and the amount of work needed to make it run. 
Therefore with all due respect the decision is yours of course. Least of all I'd want to sound demanding to people like yourself who do this work for free.

**Comment by ilteroi** [comment]
ok, fixed. interestingly, the change also makes the great wall work again if you have the "commando" promotion and move on a route. rejoice, another obscure bug squashed! should not affect performance too much either.
